src/assets/scss/main.scss

```diff
@@ -79,7 +79,11 @@ body{
 }
 
 .default-btn{
-  @apply gray-btn py-3 px-10;
+  @apply text-sm text-white font-bold transition-all duration-500 hover:bg-opacity-90 hover:shadow-lg focus:outline-none;
+}
+
+.siriusID-btn{
```

Comment:
Is this siriusID-btn class would be used elsewhere? If it is only in home vue, can declare in the vue instead of main scss file

Reply:
fixed